Tests/UnitTests/Mocks/MockOfferingsManager.swift

```diff
@@ -41,22 +43,23 @@ typealias OfferingsCompletion = @MainActor @Sendable (Result<Offerings, Error>)
         self.invokedOfferingsParametersList.append((appUserID, fetchPolicy, completion))
 
         OperationDispatcher.dispatchOnMainActor { [result = self.stubbedOfferingsCompletionResult] in
-            completion?(result!)
+            completion?(result)
         }
     }
 
     struct InvokedUpdateOfferingsCacheParameters {
         let appUserID: String
         let isAppBackgrounded: Bool
         let fetchPolicy: OfferingsManager.FetchPolicy
-        let completion: (@MainActor @Sendable (Result<Offerings, Error>) -> Void)?
```

Comment:
This wasn't used, and it meant that we could have leaked stuff inside of these closures.